golden/authorize.aes

```diff
@@ -0,0 +1,11 @@
+contract BlindAuth =
```

Comment:
please use the contract with the Auth switch https://github.com/aeternity/aepp-sdk-python/blob/10fdb242430f83c99d928e68690b630364c1b8b3/tests/test_node.py#L7

Reply:
done